src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -2451,13 +2451,14 @@ private static void ValidateEmbeddedSources_Windows(Dictionary<string, string> e
                         continue;
                     }
 
-                    var sourceStr = Encoding.UTF8.GetString(sourceBlob.Array, sourceBlob.Offset, sourceBlob.Count);
+                    // offset by 3 to skip the BOM
+                    var sourceStr = Encoding.UTF8.GetString(sourceBlob.Array, sourceBlob.Offset + 3, sourceBlob.Count - 3);
 
                     Assert.Equal(expectedEmbeddedMap[docPath], sourceStr);
                     Assert.True(expectedEmbeddedMap.Remove(docPath));
                 }
             }
-            catch
+            finally
```

Comment:
Turns out we were silently ignoring failures, but this function was not actually being used anyway :/